smokes/e2e/force.coffee

```diff
@@ -0,0 +1,63 @@
+# this file will contains the different generic functions which
+# will be called by the different tests
+# inspired by this methodology
+# http://www.lindstromhenrik.com/using-protractor-with-coffeescript/
+
+class forcePage
+    constructor: ->
+        #@loglink = element(By.id('logLink'))
+        browser.get('#/builders')
+        #return @
+
+    setReason: (reason) ->
+        reasonElement = element(By.css("forcefield label[for='reason'] + div input"))
+        expect(reasonElement.getAttribute('value')).toBe('force build')
+        reasonElement.clear()
+        reasonElement.sendKeys(reason)
+        expect(reasonElement.getAttribute('value')).toBe(reason)
+
+    setYourName: (yourName) ->
+        reasonElement = element(By.css("forcefield label[for='username'] + div input"))
```

Comment:
its not anymore the reasonElement. call it element in all the methods, and this will be fine